ulib/ml/FStar_Sealed.ml

```diff
@@ -0,0 +1,2 @@
+type 'a sealed = 'a
+let seal x = x
```

Comment:
I think we can remove this file, replaced with `ocaml/fstar-lib/FStar_Sealed.ml`